### PR TITLE
feat(github/contributors): add endpoint for repo contributors

### PR DESCRIPTION
**Pull request by sidvishnoi**
For https://github.com/w3c/respec/issues/2446

``` http
GET /github/w3c/respec/contributors
```

``` json
[
  {
    "name": "Marcos Cáceres",
    "login": "marcoscaceres",
    "contributions": 1975
  },
  {
    "name": "Robin Berjon",
    "login": "darobin",
    "contributions": 862
  },
  ...
]
```

### CODE REVIEW

File: routes/github/contributors.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// @ts-check
+const { getContributors } = require("respec-github-apis/contributors");
+const { getUsersDetails } = require("respec-github-apis/users");
+const { TTLCache } = require("respec-github-apis/utils/cache");
+
+const CACHE_TTL = 3 * 24 * 60 * 60 * 1000; // 3 days
```
Comment (sidvishnoi) [inline routes/github/contributors.js]: 3 days seems enough. Can change.

File: app.js [hunk 2]
```diff
@@ -39,6 +39,12 @@ app.post(
   "/caniuse/update",
   bodyParser.json({ verify: rawBodyParser }),
   require("./routes/caniuse/update").route
+
+app.options("/github/:org/:repo/contributors", cors({ methods: ["GET"] }));
```
Comment (sidvishnoi) [inline app.js]: can shorten it to `/gh/:org/:repo/contributors` if too long

File: package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "helmet": "^3.20.0",
     "morgan": "^1.9.1",
     "respec-caniuse-route": "^3.0.2",
+    "respec-github-apis": "github:sidvishnoi/respec-github-api",
```
Comment (sidvishnoi) [inline package.json]: will publish to npm once finalized

File: app.js (hunk 2 above)
Comment (marcoscaceres) [reply app.js]: It’s fine

File: routes/github/contributors.js (hunk 1 above)
Comment (marcoscaceres) [reply routes/github/contributors.js]: Yeah, let’s see how it goes. I don’t imagine people using this a lot (adds vanity noise to the spec).

**Pull request merged**